myhdl/_traceSignals.py

```diff
@@ -89,8 +91,13 @@ def __call__(self, dut, *args, **kwargs):
             else:
                 directory = self.directory
 
+            if self.filename is None:
+                filename = self.name
+            else
```

Comment:
missing the colon, I don't understand the description why the `name` attribute can't be use.